examples/seismic/tutorials/16_variable_z_topography.ipynb

```diff
@@ -0,0 +1,549 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Variable-z curvilinear topography\n",
```

Comment:
Hmm, a comment didn't seem to appear in my earlier review?

In any case, I think the intro needs to go through the mathematics of what is being done here in a nice clear manner. The code below can then be related back to this as appropriate.